src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -24,7 +24,28 @@ fragment ProjectOverview on Project {
       ...DisplayLocation
     }
   }
-  marketingLocation {
+  marketingCountry {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
+    }
+  }
+  marketingRegion {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
```

Comment:
Since the `marketingRegionOverride` is using `DisplayRegion`, should this be using it too instead of `DisplayLocation`?

Reply:
This is a good question.  I did try this but I would get this error message when doing so 
> Fragment "DisplayFieldRegion" cannot be spread here as objects of type "Location" can never be of type "FieldRegion". 

Only `fieldRegion` and `fieldRegionOverride` are of type "fieldRegion".  Everything else is type "Location" for now.  I believe there is a plan to move fieldRegion to a Location type but that hasn't been done yet.

Reply:
So `marketingRegion` is not region, but location?

Reply:
Our 1:1 explained why this is. All good.